OMCompiler/Compiler/Util/List.mo

```diff
@@ -284,6 +284,31 @@ algorithm
   end match;
 end isEqualOnTrue;
 
+public function allEqual<T>
+  "Takes a list and an equality function, and returns whether all elements in the list are equal."
+  input list<T> inList;
+  input CompFunc inCompFunc;
+  output Boolean outAllEqual := true;
```

Comment:
this binding has to be set with `=` rather than `:=`. locally it might work but it has been deprecated